plugins/modules/rabbitmq_upgrade.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Damian Dabrowski <damian@dabrowski.cloud>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_upgrade
+short_description: Execute rabbitmq-upgrade commands
+description:
+  - Allows to execute rabbitmq-upgrade commands
+author: "Damian Dabrowski (@damiandabrowski5)"
+version_added: '1.1.0'
+options:
+  action:
+    description:
+      - Specify action to be executed.
+    type: str
+    required: true
+    choices: ['await_online_quorum_plus_one','await_online_synchronized_mirror','post_upgrade','drain','revive']
+  node:
+    description:
+      - Erlang node name of the target rabbit node.
+    type: str
+    required: false
+    default: rabbit
+'''
+
+EXAMPLES = '''
+- name: Drain 'rabbit@node-1' node(i.e. put it into maintenance mode)
```

Comment:
```suggestion
- name: Drain 'rabbit@node-1' node (in other words, put it into maintenance mode)
```
This is a requirement of the Ansible doc style guide (we avoid using abbreviations in the doc)